### PR TITLE
Remove python-libmagic dependency

### PR DESCRIPTION
**Pull request by derek-keeler**

### CODE REVIEW

File: helpers/gogdb_helper.py [hunk 1]
```diff
@@ -2,11 +2,33 @@
...
+def is_sqlite3(path: str) -> bool:
+    """Fast check to ensure a file is indeed an SQLite3 db file, based on the format."""
+    # Thanks! https://stackoverflow.com/a/15355790/895739
+
+    test_sql_file = pathlib.Path(path)
+
+    if not test_sql_file.exists():
+        return False
+    if not test_sql_file.is_file():
+        return False
```
Comment (eniklas) [inline helpers/gogdb_helper.py:21]: `is_file` will be `False` if the file doesn't exist, so no need for the `exists` test unless I'm missing something.

```
>>> os.path.isfile("poo")
False
```

**Review by eniklas — CHANGES_REQUESTED** [review]
Just a nit about an unnecessary (I think) test, and you should remove [libmagic from the container](https://github.com/eniklas/gamatrix-gog/blob/master/Dockerfile#L5). And yay tests!

File: helpers/gogdb_helper.py (hunk 1 above)
Comment (derek-keeler) [reply helpers/gogdb_helper.py:21]: What if it is a directory...

Comment (eniklas) [reply helpers/gogdb_helper.py:21]: Then `is_file()` will be `False` as well. I'm saying to get rid of `exists()`, not `is_file()`.

Comment (derek-keeler) [reply helpers/gogdb_helper.py:21]: Ahhh... smart.

File: Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ FROM python:3.9-slim
 
 WORKDIR /usr/src/app
 
-RUN apt update && apt install -y libmagic1 && rm -rf /var/lib/apt/lists/*
+RUN apt update && rm -rf /var/lib/apt/lists/*
```
Comment (eniklas) [inline Dockerfile]: Just remove this whole line, it's not needed now.

**Review by eniklas — CHANGES_REQUESTED** [review]
Almost there, just need to ditch that whole `apt` line.

File: Dockerfile (hunk 2 above)
Comment (derek-keeler) [reply Dockerfile]: What you don't want to just randomly update the database and then delete it for no reason!? Don't you understand who I work for?

**Review by eniklas — APPROVED** [review]
Nice one dude, merge away!

**Pull request merged**